extensions/applicationinsights-properties-js/Tests/Unit/src/properties.tests.ts

```diff
@@ -624,14 +624,13 @@ export class PropertiesTests extends AITestClass {
             name: "Storage Prefix Test for Property Plugin: prefix should be added after init",
             useFakeTimers: true,
             test: () => {
-                let core = new AppInsightsCore();
                 let setItemSpy = this.sandbox.spy(window.localStorage, "setItem");
                 let storagePrefix = "storageTestPrefix"
                 let coreConfig = {
                     instrumentationKey: "b7170927-2d1c-44f1-acec-59f4e1751c13ttt",
                     storagePrefix: storagePrefix,
                 }
-                core.initialize(coreConfig, [this.properties])
+                this.properties.initialize(coreConfig, this.core, []);
```

Comment:
Why change?

It would be better to keep initializing from the core like the original...

Reply:
because in the main branch, properties.test does not have the fakeChannel, which cause core initilize not working